src/DataLayer/converters/MeasureConverter.ts

```diff
@@ -166,6 +166,12 @@ function getFormat(measure: IMeasure): string | undefined {
     const { measure: { definition } } = measure;
     const measureFormat = get(measure.measure, 'format');
 
+    if (VisualizationObject.isArithmeticMeasureDefinition(definition)) {
+        if (definition.arithmeticMeasure.operator === 'change') {
+            return '#,##0.00%';
```

Comment:
I don't know how's that possible, but it works also for localized number formatting